_components/search-bar.md

```diff
@@ -9,11 +9,13 @@ lead: Intro text on what is included in this section and how to use it. No more
   <div class="usa-grid">
     <div class="usa-width-one-half">
       <form class="usa-search usa-search-big">
-        <label for="search-field-big">Search Big</label>
-        <input type="search" id="search-field-big">
-        <button type="submit">
-          <span class="usa-search-submit-text">Search</span>
-        </button>
+        <div role="search">
```

Comment:
@msecret I'm ok with using that, but from the view of someone looking to use this component, they'd see this: `<div class="usa-width-one-half" role="search">` and I'm worried they'd think they'd _always_ need to use a wrapper div. We could put this information in the documentation, but for someone that wants to "grab and go" I'm worried this will be lost.